docs/core/whats-new/dotnet-8/overview.md

```diff
@@ -21,6 +21,10 @@ For information about what's new in the .NET SDK, code analysis, and diagnostics
 
 C# 12 shipped with the .NET 8 SDK. For more information, see [What's new in C# 12](../../../csharp/whats-new/csharp-12.md).
 
+## F# 8
+
+F# 8 focuses on making the language simpler, more consistent, and more performant. For more information, see [What's new in F# 8](../../../fsharp/whats-new/fsharp-8.md).
+
```

Comment:
This brings no value, this should either some details, be consistent with C# 12 above, or not be here whatsoever.